tests/test_core.py

```diff
@@ -5,17 +5,17 @@
 import shutil
 import tempfile
 import unittest
+import pytest
 
 import opentimelineio as otio
 
 
 class TestCoreFunctions(unittest.TestCase):
```

Comment:
Since we're converting to pytest, this no longer needs to be a subclass of `unittest.TestCase`, it can be converted to:
```python
class TestCoreFunctions:
```